compiler/compile.bal

```diff
@@ -68,7 +72,9 @@ class CompileContext {
 function compileBalFile(string filename, string basename, string? outputBasename, nback:Options nbackOptions, OutputOptions outOptions) returns CompileError? {
     CompileContext cx = new(basename, outputBasename, nbackOptions, outOptions);
     front:ResolvedModule mod = check processModule(cx, DEFAULT_ROOT_MODULE_ID, [ {filename} ], cx.outputFilename());
+    // pr-todo: find if this should be moved to init.bal
```

Comment:
No, because nback doesn't depend on front.

Reply:
I didn't mean just the method call, I mean move the whole logic to `init.bal`.

Reply:
Discussed in the meeting, this logic belongs in the frontend.